### DISCUSSION TITLE
How to draw an IPF color characterization map using MTEX Matlab

### DISCUSSION THREAD

**Discussion by qiqianhello** [mtex-toolbox/mtex#1246] [Ask Anything] [answered]
Hi, I try to draw an IPF colour characterization map with [100], [101] and [111] using MTEX Matlab. But I do not know the code. Can you help me? Thanks.

**Comment by kilir** [comment, marked as answer]
Hi,
I think you need to be a little more verbose about what you want to achieve.
Cheers,Rüdiger

**Reply by qiqianhello** [reply]
Thanks.
I am new to Mtex.
I got my EBSD data. The vendor just provide the IPF key map with [100], [001] and [110]. I want to change them to [001], [101] and [111].
Please see the attached figure 1. 
While, what I actually want is that similar to Figure 2.
Thanks.
![Figure 1](https://user-images.githubusercontent.com/55877865/136180795-8cfdf7a9-4a51-488d-a430-c8b3353be2bd.jpg)
![Figure 2](https://user-images.githubusercontent.com/55877865/136181081-cb5c7582-594f-481f-9981-659f11a8086e.jpg)

.
